### PR TITLE
README.md updated: Spark 2.x Syntax and Typo resolved

### PR DESCRIPTION
**Pull request by chetkhatri**
## What changes were proposed in this pull request?

* Spark 2.x syntax replaced with Spark 1.x syntax
* Typo Fixed.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,8 +1,8 @@
 # Apache Spark - Apache HBase Connector
 
-The [Apache Spark](https://spark.apache.org/) - [Apache HBase](https://hbase.apache.org/) Connector is a library to support Spark accessing HBase table as external data source or sink. With it, user can operate HBase with Spark-SQL on data frame level.
+The [Apache Spark](https://spark.apache.org/) - [Apache HBase](https://hbase.apache.org/) Connector is a library to support Spark accessing HBase table as external data source or sink. With it, user can operate HBase with Spark-SQL on data frame,DataSet level.
```
Comment (weiqingy) [inline README.md]: A space between "`data frame,`" and "`DataSet`"?

**Comment by weiqingy** [comment]
Thanks, @chetkhatri . I'll merge it to Master.

File: README.md [hunk 2]
```diff
@@ -29,7 +29,7 @@ Note that the WHERE conditions need to be defined carefully. Otherwise, the resu
 Both are exposed to users by specifying WHERE CLAUSE, e.g., where column > x and column < y for scan and where column = x for get. All the operations are performed in the executors, and driver only constructs these operations. Internally we will convert them to scan or get or combination of both, which return Iterator[Row] to catalyst engine. 
 
 ## 
-Creatable DataSource  The libary support both read/write from/to HBase.
```
Comment (weiqingy) [inline README.md]: Actually, there was a format mistake. 
```
 ## 
 Creatable DataSource  The libary support both read/write from/to HBase. 
```
should be 
```
## Creatable DataSource  
The libary support both read/write from/to HBase. 
```

File: README.md (hunk 1 above)
Comment (chetkhatri) [inline README.md]: Fixed that

**Comment by chetkhatri** [comment]
@weiqingy  Sure, I have committed. could you please check it, if  everything seems good merge it.

**Comment by chetkhatri** [comment]
@weiqingy  Don't you think giving constant values in spark-submit command in example is also not best practice. such as, --num-executors 2 --driver-memory 512m --executor-memory 512m --executor-cores 1 in README.md , if you confirm i can do change that.

**Comment by weiqingy** [comment]
@chetkhatri Yes. Let's remove those constants.

**Comment by chetkhatri** [comment]
@weiqingy Could you please review updated PR ?

File: README.md [hunk 3]
```diff
@@ -1,8 +1,8 @@
 # Apache Spark - Apache HBase Connector
 
-The [Apache Spark](https://spark.apache.org/) - [Apache HBase](https://hbase.apache.org/) Connector is a library to support Spark accessing HBase table as external data source or sink. With it, user can operate HBase with Spark-SQL on data frame level.
+The [Apache Spark](https://spark.apache.org/) - [Apache HBase](https://hbase.apache.org/) Connector is a library to support Spark accessing HBase table as external data source or sink. With it, user can operate HBase with Spark-SQL on DataFrame and DataSet level.
 
-With the data frame support, the lib leverages all the optimization techniques in catalyst, and achieves data locality, partition pruning, predicate pushdown, Scanning and BulkGet, etc.
+With the DataFrame and DataSet support, the lib leverages all the optimization techniques in catalyst, and achieves data locality, partition pruning, predicate pushdown, Scanning and BulkGet, etc.
 
 _**Note**: Master branch matches Spark 2.1.x. com.hortonworks:shc-core:1.1.0-2.1-s_2.11 and com.hortonworks:shc-examples:1.1.0-2.1-s_2.11 have not been uploaded to Hortonworks public repo, but will be there soon_.
```
Comment (weiqingy) [inline README.md]: "com.hortonworks:shc-core:1.1.0-2.1-s_2.11 and com.hortonworks:shc-examples:1.1.0-2.1-s_2.11" have been uploaded in Hortonworks public repo. So we can also remove "com.hortonworks:shc-core:1.1.0-2.1-s_2.11 and com.hortonworks:shc-examples:1.1.0-2.1-s_2.11 have not been uploaded to Hortonworks public repo, but will be there soon"

Comment (chetkhatri) [inline README.md]: @weiqingy Changed committed.

Comment (weiqingy) [inline README.md]: @chetkhatri Thanks for the quick response, but it seems the new commit changed a different place?

**Comment by chetkhatri** [comment]
@weiqingy Can I assume, it were expected to remove from both places. check recent commit.

File: README.md [hunk 4]
```diff
@@ -115,21 +115,19 @@ Given a data frame with specified schema, above will create an HBase table with
...
     val df = withCatalog(catalog)
     //SQL example
-    df.registerTempTable("table")
+    df.createOrReplaceTempView("table")
     sqlContext.sql("select count(col1) from table").show
 
 ## Configuring Spark-package
 Users can use the Spark-on-HBase connector as a standard Spark package. To include the package in your Spark application use:
 
-_**Note**: com.hortonworks:shc-core:1.1.0-2.1-s_2.11 has not been uploaded to [spark-packages.org](https://spark-packages.org/package/hortonworks-spark/shc), but will be there soon._
```
Comment (weiqingy) [inline README.md]: We can just leave this line here since we have not updated SHC packages to spark-packages.org.

File: README.md (hunk 3 above)
Comment (chetkhatri) [inline README.md]: @weiqingy Oh, I see my bad.

**Comment by weiqingy** [comment]
Thanks, @chetkhatri . Merged to Master.

**Pull request merged**